src/utils/httpUtils.js

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
just make this a single-purpose top-level module called bulkUpsert.js 

```
module.exports = (refocusInstance, arr) => {
  ...
}
```

Reply:
We can add doPost, doGet method as well over here When I will do new register flow. @iamigo

Reply:
OK, makes sense. Go ahead.